packages/cli/src/Generate.ts

```diff
@@ -98,8 +96,8 @@ ${bold('Examples')}
       '-h': '--help',
       '--watch': Boolean,
       '--schema': String,
-      '--data-proxy': Boolean,
-      '--accelerate': Boolean,
+      '--data-proxy': Boolean, // remove in Prisma 6
+      '--accelerate': Boolean, // remove in Prisma 6
```

Comment:
Still exists for backwards compatibility but does nothing. The switch is now dynamic and Data Proxy is loaded based on the URL.

Reply:
Removing it later without a warning before does not sound "great" 🤔
I can see us potentially keeping this in Prisma 6 to avoid a "useless" breaking change.